<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Pyramid images</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>39</number>
   </property>
   <item row="0" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxCoefficient">
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>582</width>
       <height>426</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>582</width>
       <height>426</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>582</width>
       <height>426</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>580</width>
        <height>424</height>
       </rect>
      </property>
      <widget class="QLabel" name="labelImage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>581</width>
         <height>451</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>581</width>
         <height>451</height>
        </size>
       </property>
       <property name="text">
        <string>Изображение</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxImage">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="comboBoxLayer"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelSize">
     <property name="text">
      <string>0x0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>comboBoxLayer</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Widget</receiver>
   <slot>setCurrentIndexLayer(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxCoefficient</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Widget</receiver>
   <slot>setCoefficient(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Widget</sender>
   <signal>labelSizeChanged(QString)</signal>
   <receiver>labelSize</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxImage</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Widget</receiver>
   <slot>setCurrentIndexImage(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Widget</sender>
   <signal>labelImageChanged(QPixmap)</signal>
   <receiver>labelImage</receiver>
   <slot>setPixmap(QPixmap)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Widget</sender>
   <signal>comboBoxImageEnable(bool)</signal>
   <receiver>comboBoxImage</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>labelSizeChanged(QString)</signal>
  <signal>labelImageChanged(QPixmap)</signal>
  <signal>comboBoxImageEnable(bool)</signal>
  <slot>setCurrentIndexImage(int)</slot>
  <slot>setCurrentIndexLayer(int)</slot>
  <slot>setCoefficient(double)</slot>
 </slots>
</ui>
